### PR TITLE
Merging wiki into a docs folder

### PR DESCRIPTION
**Pull request by jimangel**
This PR pulls all of the wiki content into a traditional /docs folder. I've also began to clean up our main README.md page and fix any links referencing the wiki.

Merging this PR will be the foundation to continue overhauling our docs. I've created a to do list for my next action items: https://github.com/kubernetes/dashboard/issues/3172

Preview: https://github.com/jimangel/dashboard/tree/docs-refresh

Note: Once merged, please delete the repo's wiki to avoid confusion. Only repo owners can delete the wiki.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,28 +36,30 @@ Now access Dashboard at:
...
 * The shortcut `http://localhost:8001/ui` is deprecated. Use the full proxy URL shown above.
 * [Heapster](https://github.com/kubernetes/heapster/) has to be running in the cluster for the metrics
-and graphs to be available. Read more about it in [Integrations](
-https://github.com/kubernetes/dashboard/wiki/Integrations) guide.
+and graphs to be available. Read more about it in [Integrations](docs/contributors/Integrations) guide.
 
 ## Documentation
 
-Dashboard documentation can be found on [Wiki](https://github.com/kubernetes/dashboard/wiki) pages, it includes:
+(content in progress)
```
Comment (jeefy) [inline README.md]: This probably shouldn't be in progress when we merge into master.

File: docs/README.md [hunk 2]
```diff
@@ -1,4 +1,45 @@
-# Dashboard Documentation
+## Advanced Topics and Tutorials
```
Comment (jeefy) [inline docs/README.md]: Might be nit-picky but I wouldn't consider the FAQ an advanced topic. Maybe title it "Additional topics and documentation"?

File: README.md (hunk 1 above)
Comment (jimangel) [reply README.md]: Good catch, I can just pull this for the initial merge. My goal was to get a foundation in place then start the clean up as long as we're all aligned on direction.

File: docs/README.md (hunk 2 above)
Comment (jimangel) [reply docs/README.md]: Not nit-picky at all, I took a lot of design inspiration from https://github.com/kubernetes/minikube but I agree that FAQ doesn't fall under advanced :) - I'll make those changes.

**Comment by jimangel** [comment]
Closing per discussion about opening up wiki, will update readme and wiki moving forward.
/close

**Pull request closed without merging**